README.md

```diff
@@ -48,7 +47,7 @@ our documentation, add new features—up to you! Thank you for pitching in.
 
 ## Compatibility
 
-Mail supports Ruby 1.8.7+, including JRuby and Rubinius.
+Mail supports Ruby 2.5+, including JRuby and Rubinius.
```

Comment:
I think Rubinius is gone (at least at CI level). We can swap it with TruffleRuby.

```suggestion
Mail supports Ruby 2.5+, including JRuby and TruffleRuby.
```